controllers/storagecluster/cephcluster.go

```diff
@@ -110,6 +130,37 @@ func (obj *ocsCephCluster) ensureCreated(r *StorageClusterReconciler, sc *ocsv1.
 		return reconcile.Result{}, fmt.Errorf("'StorageDeviceSets' should not be initialized in an external CephCluster")
 	}
 
+	// remove the fix in the next release 4.19
```

Comment:
better move all new code to a helper function.  The existing `ensureCreated` is already too big.